server/.env.example

```
@@ -0,0 +1,23 @@
+# Akash VPN Server Configuration
+# Copy this file to .env and fill in your actual VPN server credentials
+
+# Belgium Server Configuration
+BELGIUM_HOSTNAME=belgium-hostname.example.com
+BELGIUM_PORT=32576
+BELGIUM_USERNAME=belgium-username
+BELGIUM_PASSWORD=belgium-password
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The BELGIUM_PASSWORD key should go before the BELGIUM_PORT key

(UnorderedKey)